### PR TITLE
Apply copier template

### LINKED ISSUE #24: Move to use `scipp/copier_template`

### PR DESCRIPTION
**Pull request by jl-wynen**
Fixes #24

### CODE REVIEW

**Comment by jl-wynen** [comment]
Note that there are placeholders for the Zenodo links. Those can only be filled in when we make a new release. See #26

File: requirements/docs.in [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-ipywidgets
-matplotlib
+myst-parser
 nbsphinx
 plopp
+pydata-sphinx-theme>=0.13
 sphinx
-sphinx-autodoc-typehints==1.18.3  # later versions require sphinx >=5 which is incompatible with current sphinx-book-theme
-sphinx-book-theme==0.3.3
+sphinx-autodoc-typehints==1.23.0  # Higher versions require sphinx-7, which is not supported by sphinx-design yet
```
Comment (nvaytet) [inline requirements/docs.in:10]: cf your other PR, has this not been fixed now?

File: requirements/nightly.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Generated by 'tox -e deps', DO NOT EDIT MANUALLY!'
+https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (nvaytet) [inline requirements/nightly.in:2]: `cp38-cp38`: has something gone wrong in the name here?

**Review by nvaytet — COMMENTED** [review]
Looks good overall.

File: requirements/docs.in (hunk 1 above)
Comment (jl-wynen) [reply requirements/docs.in:10]: Yes. But this was not in the template when I applied it here. I can wait with merging until the changes in the template are in.

File: requirements/nightly.in (hunk 2 above)
Comment (jl-wynen) [reply requirements/nightly.in:2]: This is how it's supposed to be. See here: https://pypi.org/project/scipp/#files
I don't know why it's repeated.

**Review by nvaytet — APPROVED** [review]
> Yes. But this was not in the template when I applied it here. I can wait with merging until the changes in the template are in.

Do what you like. Either waiting now, or running a copier update later is equivalent.

**Pull request merged**